manala.composer/tests/0110_version.goss.yml

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
As it's part of the `install` task could you rename the test to `0110_install_version.yml` ?